### ISSUE TITLE
Feature: Display image dimensions for AVIF files in the tooltip

### ISSUE THREAD

**Issue by x-rated** [files-community/Files#16289]
### Description

size of image in tooltip is missing

![tooltip](https://github.com/user-attachments/assets/6d3d02b2-b106-4767-afa1-d94bd556cebf)

### Steps To Reproduce

hover mouse over some avif image

### Requirements

there should be image dimensions as shown here https://files.community/blog/posts/v3-5#view-dimensions-when-hovering-over-images

### Files Version

3.7.7.0

### Windows Version

26120.1912

### Log File

[debug.log](https://github.com/user-attachments/files/17211358/debug.log)

**Comment by yaira2** [comment]
It seems that the API we're using doesn't work for AVIF files. It can be done using a different API but we'll need to redo the code first.

**Comment by yaira2** [comment]
@0x5bfa we might want to consider using the same system that we have in place for the details pane.

**Comment by 0x5bfa** [comment]
If file Explorer can display by default there should be a property key for it.
I suspect [System.Image.Dimentions](https://learn.microsoft.com/en-us/windows/win32/properties/props-system-image-dimensions) is what it is. So we might want to use this for all images at the end of the day.

@x-rated it can for AVIF?

**Comment by yaira2** [comment]
> I suspect [System.Image.Dimentions](https://learn.microsoft.com/en-us/windows/win32/properties/props-system-image-dimensions) is what it is. So we might want to use this for all images at the end of the day.

Correct, we already use this in the details pane. We should be able to do the same for the tooltip.

**Comment by x-rated** [comment]
@0x5bfa windows explorer can show the dimensions 🤔

![image](https://github.com/user-attachments/assets/9f83f9fb-b6a7-481c-9904-819b59360c47)

**Comment by 0x5bfa** [comment]
That is good to hear. Easy one.

**Comment by yaira2** [comment]
@x-rated this feature was added in a recent preview build (v3.8.13) and will be included in the next release.

https://files.community/blog/posts/v3-8-13
